### PR TITLE
Way to many small edits (and a title suggestion)

### PR DESCRIPTION
**Pull request by tpoisot**
I added "digital" in the title, because this is not a paper about safely storing your bound notebooks.

Oh, can we write this one too?

### CODE REVIEW

**Comment by tpoisot** [comment]
And of course there is a typo in the title of this pull request :clap:

File: manuscript/manuscript.md [hunk 1]
```diff
@@ -195,67 +178,57 @@ ideally prior to beginning data collection:
    tracked changes be logged?
  - Will the final data be released, and if so, in what format?
  - Are there restrictions or privacy concerns associated with the data
-   (e.g. for survey results)?
+   (e.g. for survey results, threatened species)?
  - Will institutional validation be required prior to releasing the
    data?
- - Does the agency funding the research require data deposition, and
+ - Does the funding agency the research mandates data deposition, and
```
Comment (jhollist) [inline manuscript/manuscript.md:184]: Think it should read
- Does the funding agency mandate data deposition, and

File: manuscript/manuscript.md [hunk 2]
```diff
@@ -195,67 +178,57 @@ ideally prior to beginning data collection:
...
    if so, where?
- - Does the journal in which you plan to publish require data
-   deposition?
-
-<!-- PB comment: we should probably make a decision whether to use
-you/yours or not. Some rules do and some don't. -->
-<!-- NZ comment: I reordered these to make better logical sense and
-   tried to remove 2nd person where possible -->
+ - Does the target journal mandates data deposition?
```
Comment (jhollist) [inline manuscript/manuscript.md:186]: singular on mandate

File: manuscript/manuscript.md [hunk 3]
```diff
@@ -195,67 +178,57 @@ ideally prior to beginning data collection:
...
 
-None of these questions have universal answers, nor are they the only
-questions one should ask before starting data acquisition. But
-similarly to Rule 1, knowing the what, when, and how of *your* use of
-the data will bring you close to a reliable roadmap on how to handle
-data from acquisition through publication to archival.
+None of these questions have universal answers, nor are they the only questions
+one should ask before starting data acquisition. But as for Rule 1, knowing the
```
Comment (jhollist) [inline manuscript/manuscript.md:189]: Should be: But, as for Rule 1, knowing...

File: manuscript/manuscript.md [hunk 4]
```diff
@@ -195,67 +178,57 @@ ideally prior to beginning data collection:
...
+what, when, and how of *your* use of the data will bring you close to a reliable
+roadmap on how to handle data from acquisition through publication to archival.
```
Comment (jhollist) [inline manuscript/manuscript.md:191]: Not sure on this but maybe should just be "archive" and not "archival"

**Comment by jhollist** [comment]
Made a few editorial comments.  Once those are addressed, I think this PR is ready to merge.

**Pull request merged**